website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md

```diff
@@ -44,7 +44,7 @@ To improve your experience using dbt Cloud, we suggest that you turn off ad bloc
 
 ## dbt Cloud IDE features
 
-The dbt Cloud IDE comes with [tips](/docs/cloud/dbt-cloud-ide/dbt-cloud-tips) and [features](/docs/cloud/dbt-cloud-ide/ide-user-interface) that make it easier for you to develop, build, compile, run, and test data models. 
+The dbt Cloud IDE comes with [tips](/docs/build/dbt-tips) and [features](/docs/cloud/dbt-cloud-ide/ide-user-interface) that make it easier for you to develop, build, compile, run, and test data models. 
```

Comment:
There is nothing on the tips page that points out anything related to the IDE. Maybe this should be the keyboard shortcuts link?

Reply:
ive tweaked this too but the features table does have the keyboard shortcut on the first line and the tips page does have the ide shortcuts linked -- wondering how come it didn't show up for you? 
![Screenshot 2023-12-18 at 12 08 08](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/69663197-de54-4c56-ac1d-55aed175dbcc)